config/routes.rb

```diff
@@ -42,6 +42,9 @@
     get "/identity/simulate", to: "identity#new"
     get "/identity/users", to: "users#index"
     get "/identity/users/:uuid", to: "users#show", as: :identity_user
+    get "/identity/users/update/:uuid",
+        to: "users#update",
+        as: :identity_user_update
```

Comment:
This PR is a temporary implementation, so this can probably go in, but I want to point out that this action shouldn't be a GET because it's not idempotent and causes a side effect

Reply:
Thanks @tvararu , yes this was just temporary, the plan is to change it to a post request for https://trello.com/c/CPLxwXqk